docs/changelog.md

```diff
@@ -35,13 +35,24 @@ permalink: /changelog/
   * Added [BrowserMenuImageText](https://github.com/mozilla-mobile/android-components/blob/master/components/browser/menu/README.md#BrowserMenuImageText) for show an icon next to text in menus.
   * Added support for showing a menu with DOWN and UP orientation (e.g. for supporting menus in bottom toolbars).
 
-* **concept-engine**, **browser-engine-gecko-***
+* **concept-engine**, **browser-engine-gecko-**
   * Added support for enabling tracking protection for specific session type:
   ```kotlin
   val engine = GeckoEngine(runtime, DefaultSettings(
     trackingProtectionPolicy = TrackingProtectionPolicy.all().forPrivateSessionsOnly())
   )
   ```
+* **support-ktx**
+  * Added `Lifecycle.addObservers` to observe the lifecycle for multiple classes.
+  ```kotlin
+  override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+    lifecycle.addObservers(
+      fullscreenFeature,
+      sessionFeature,
+      customTabsToolbarFeature
+    )
+  }
+  ```
```

Comment:
This is the wrong version - this should go in 0.40.0. :)

Reply:
Oops! Thanks for the catch!